src/endpoints/dockerflow.rs

```diff
@@ -26,7 +26,7 @@ pub fn heartbeat(app_data: Data<EndpointState>) -> Result<HttpResponse, Classify
             Some(country_info) => country_info
                 .country
                 .and_then(|country| country.iso_code)
-                .and_then(|iso_code| Some(Ok(iso_code == "US")))
+                .and_then(|iso_code| Some(Ok(iso_code.len() > 0)))
```

Comment:
Clippy wants this to be

```suggestion
                .and_then(|iso_code| Some(Ok(!iso_code.is_empty())))
```